src/cleanup_rules/swift/rules.toml

```diff
@@ -250,7 +250,8 @@ name = "if_always_true"
 query = """ (
 (if_statement
     condition: (boolean_literal) @condition_literal
-    .(statements) @if_block
+    ((comment)*
```

Comment:
how about just `(_)*`

Reply:
We want to capture the first (statements) block separately. If we do just (_)* we will catch all the if else cases since else and else if are also considered as children of the first if.

Reply:
But then this does not seem a sound rule either. How abut altering the grammar for swift to fix these `if-else` awkwardness from the root ?

Reply:
Waiting for that sounds a little unreasonable to me. Also, we would need to confirm with @Satyam1749 because if I remember he wanted to pick that up but it was an optional improvement.

Reply:
Its alright if you dont want to take this up, we will not land this either . When I get time i will look into it

Reply:
How about we merge this while we look to solve that? Once solved, we can revert this change because if we solve grammar a lot of rules will have to be re-written anyway. Especially these if else cleanups.

Reply:
@ketkarameya `(comments)* .(statements)` is a full proof way to handle if-blocks with current grammar and there can't be an edge case to it. So, in my opinion we should go ahead with this for now. Since, fixing the tree-sitter grammar is a 3rd party dependency. 

This usecase is a pretty basic usecase so we shouldn't keep this blocked on 3rd party dependency.